src/ai/backend/storage/api/manager.py

```diff
@@ -168,13 +170,15 @@ class Params(TypedDict):
                     t.Key("qsid"): tx.QuotaScopeID(),
                     t.Key("options", default=None): t.Null | QuotaConfig.as_trafaret(),
                 },
-            ),
+            ).allow_extra("*"),
```

Comment:
I think it would be much more better to explicitly define nullable `extra_args` unless there is a specific reason not to do so.